.gitlab-ci.yml

```diff
@@ -25,6 +26,40 @@ builder-image:
     - docker build --cache-from $CI_REGISTRY_IMAGE:builder-$CI_COMMIT_REF_SLUG --cache-from $CI_REGISTRY_IMAGE:builder-develop -t $CI_REGISTRY_IMAGE:builder-$CI_COMMIT_REF_SLUG -f Dockerfile.builder .
     - docker push $CI_REGISTRY_IMAGE:builder-$CI_COMMIT_REF_SLUG
 
+.build-depends-template:
+  stage: build-depends
+  image: $CI_REGISTRY_IMAGE:builder-$CI_COMMIT_REF_SLUG
+  variables:
+    SDK_URL: https://bitcoincore.org/depends-sources/sdks
+    OSX_SDK: "10.11"
+    MAKEJOBS: -j4
```

Comment:
In commit 04372dd95f21c62095e467d6754d79bc8b4035a4: why using these variables here and in multiple cases below and not `source ./ci/matrix.sh`?

Reply:
Because building of dependencies is not based on BUILD_TARGET anymore, so `matrix.sh` wouldn't work anymore. Dependencies are now built per HOST (twice for `x86_64-unknown-linux-gnu` to get a debug version). If we want to stay with Travis forever, I'd suggest to change the Travis build as well to be more like this.